src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -339,24 +339,12 @@
   <data name="Globals_WordDelimiters.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
     <value>Determines the delimiters used in a double click selection.</value>
   </data>
-  <data name="Nav_AddNewProfile.Content" xml:space="preserve">
-    <value>Add new</value>
-  </data>
-  <data name="Nav_Appearance.Content" xml:space="preserve">
-    <value>Appearance</value>
-  </data>
   <data name="Nav_AppearanceGlobal.Content" xml:space="preserve">
-    <value>Global</value>
+    <value>Appearance</value>
   </data>
   <data name="Nav_ColorSchemes.Content" xml:space="preserve">

```

Comment:
we already got `NavAppearance` translated for `Appearance`. I'd rather we do not change the _contents_. If possible, we should move any previous use of `Nav_AppearanceGlobal` to `Nav_Appearance` instead.